identity/README.md

```diff
@@ -8,27 +8,35 @@ such as users, roles, permissions, and OAuth clients.
 
 ## Running locally
 
-1. Build project from the root folder of the monorepo using:
+- All commands should be run from the root folder of the monorepo unless stated otherwise.
+
+1. Build project:
 
 ```
-mvn clean install -U --projects dist --also-make -DskipTests -Dskip.fe.build=false
+mvn clean install -DskipTests=true -DskipChecks -Dskip.fe.build=false -DskipQaBuild -T1C
 ```
 
 2. Run elasticsearch by using one of the already configured docker-compose file in our codebase.
 
-Example: navigate to Operate's folder in the codebase and run:
+Example (using Operate's configuration):
 
 ```
-docker compose -f config/docker-compose.yml up -d elasticsearch
+docker compose -f operate/docker-compose.yml up -d elasticsearch
```

Comment:
@vitorwtavares Can't we create a docker compose for Identity?

Reply:
@vsgoulart That would be good, but I have very little context of the configuration around the Identity project, so I'd rather leave that task for someone else on the team with more context and docker knowledge if possible. 

This change is mostly intended so we don't cause unexpected blockers for whoever's trying to run the project by consulting this README, not really focused on optimizing the workflow.

If by any chance you know what configuration is and is not needed for an Identity docker compose file, feel free to add it as part of this PR or in a separate one.

Reply:
I created this issue for this to not lose track of it:
https://github.com/camunda/camunda/issues/29205

Reply:
I created this issue to not lose track of the above question
#29205